shared/src/commonMain/kotlin/dev/johnoreilly/confetti/ConfettiRepository.kt

```diff
@@ -86,11 +86,11 @@ class ConfettiRepository : KoinComponent {
         uid: String?,
         tokenProvider: TokenProvider?,
         fetchPolicy: FetchPolicy
-    ): ApolloResponse<GetBookmarksQuery.Data> =
+    ): Flow<ApolloResponse<GetBookmarksQuery.Data>> =
         apolloClientCache.getClient(conference, uid).query(GetBookmarksQuery())
             .tokenProvider(tokenProvider)
             .fetchPolicy(fetchPolicy)
-            .execute()
+            .toFlow()
```

Comment:
This can't be execute if we take fetchPolicy in as a parameter.

Reply:
Agreed. `execute()` is a bit awkward because of `CacheAndNetwork` that emits twice. There is a bit of brain muscle around it though so it's hard to change at this point.

Reply:
I might add a simple one so it's hidden from clients.

Reply:
Probably an extension function to ApolloCall?

Reply:
Not sure I follow. Soemthing like this?

```
ApolloCall.cacheAndNework()
```

Reply:
Not sure, I think I want `ApolloCall.asStateFlow(fetchPolicy): StateFlow<Result<Data>>` that hides all errors when it has any results to show. And a failed form if both network and cache failed.

Reply:
Hiding errors is a bit risky. There's some background there where we did that for `.watch()` and some users required access to the individual errors.

Reply:
What about Flow<ApolloResponse> but it doesn't go from a a non null data, to null?

Reply:
Maybe yea. In all cases, I'd proceed with caution there as we've been burnt by this before (one instance [there](https://github.com/apollographql/apollo-kotlin/issues/3976)). Maybe keep the current `toFlow()` that emits everything and add helper functions on top of it. If it's `StateFlow` it can be one layer of abstraction above or so.